tests_bucklescript/operations/fragmentDefinition.re

```diff
@@ -1,9 +1,15 @@
+// TODO: we're flattening module when there is only one fragment. This seems misleading
```

Comment:
You can do the following: 

```reason
module Fragments = {
  [%graphql
  {|
    fragment ListFragment on Lists {
      nullableOfNullable
      nullableOfNonNullable
    }
  |}]
}
```

Reply:
I think I changed this because it is confusing if `%graphql` works different for fragments than operations. I think this notation is best used when you'd like to rename the query or fragment in reason, but generally we should encourage the notation without `module X =`.

Reply:
I have mixed feelings about it. I really like `module X = `we have right now and I'm not sure if it's a good idea to default to syntax without it.

Reply:
I think it's better to have syntax that behaves the same for operations and fragments, otherwise it's very confusing for users. And the difference is very small, just add two brackets. Changing the syntax for operations is also not great.